packages/plugin-core/src/test/suite-integ/BacklinksTreeDataProvider.test.ts

```diff
@@ -126,13 +130,6 @@ suite("BacklinksTreeDataProvider", function () {
           ?.enableUnrefLinks;
         expect(isUnrefLinkEnabled).toBeTruthy();
 
-        // currently unref links are not getting populated on the first engine init for some reason.
-        // FIX: once above problem is fixed, fix this test.
-        await NOTE_PRESETS_V4.NOTE_WITH_UNREF_TARGET.create({
-          wsRoot,
-          vault: vaults[0],
-        });
-        // TEMP: need to re-init.
         await new ReloadIndexCommand().execute();
```

Comment:
do we still need a reloadindex command here?

Reply:
yes, unlink refs would not have been added because we waited to set the `enableUnlinkRef` flag to true until this point. We need to reload the index so it does it for the first time.